docs/modules/ROOT/pages/index.adoc

```diff
@@ -10,7 +10,13 @@ Live code the backend and frontend together with close to no configuration. When
 
 * Create or use an existing Quarkus application
 * Add the Quinoa extension
-* Install NodeJS (https://nodejs.org/) or make sure Quinoa is link:config-reference.adoc#quarkus-quinoa_quarkus.quinoa.package-manager-install[configured] to install it.
+* Install NodeJS (https://nodejs.org/) or make sure Quinoa is https://docs.quarkiverse.io/quarkus-quinoa/dev/config-reference.html#quarkus-quinoa_quarkus.quinoa.package-manager-install[configured] to install it, see the following example.
+
+----
+quarkus.quinoa.package-manager-install=true
```

Comment:
You don't need the `install=true` step anymore in 2.2.1 we detect any change in package.json and we automatically run on the first instance.

Reply:
Tested it on Horreum and worked fine. Thanks :)